website/docs/validator.md

```diff
@@ -42,4 +42,33 @@ for _, errs := range validationErr {
 		t.Errorf("%s: cmd %s: %s", errs.Rule, errs.Cmd.CommandPath(), errs.Name)
 	}
 }
-```
\ No newline at end of file
+```
+
+## Ignore Commands
+Sometimes during development, you want to pass the tests for certain commands, but at the same time use Validator for tests. Validation can be skipped/ignored for the commands, mentioned in the validator configuration.
+To ignore the commands you need to specify the path of the command in validator configuration.
+
+ Skip single command `mycli actions create`
+```go
+ValidatorOptions: rules.ValidatorOptions{
+	SkipCommands: map[string]bool{"mycli actions create": true},
+},
+```
+
+2. Skip the command including all children
+```go
+ValidatorOptions: rules.ValidatorOptions{
+	SkipChildren: map[string]bool{"mycli": true},
+},
+```
+
+3. Skip the command for specific rule
+```go
+Length: rules.Length{
+	RuleOptions: validator.RuleOptions{
+		SkipCommands: map[string]bool{"mycli actions create": true},
```

Comment:
@ankithans is that ok?

Reply:
yes.. this looks good!

Reply:
example had only `create` command so I wasn't sure if you are using cobra paths for this